### PR TITLE
cleanup pylintrc

### PR DESCRIPTION
**Pull request by ksivaman**
Cleanup pylint config file after removal of PyTorch softmax extensions

Signed-off-by: Kirthi Shankar Sivamani <ksivamani@nvidia.com>

### CODE REVIEW

**Review by ptrendx — APPROVED** [review]
LGTM
